src/pyvoices/commands/chunk.py

```diff
@@ -40,13 +42,12 @@ def run(args):
         if concept in intervalByConcept:
             # We can match a transcription to a lable in the Praat file, ...
             interval = intervalByConcept[concept]
-            # .. so we can cut out the corresponding audio chunk ...
-            audio_chunk = audio[interval.start * 1000 - 30:interval.end * 1000 + 30]
-            # ... make it smooth around the edges ...
-            audio_chunk.fade_in(30)
-            audio_chunk.fade_out(30)
-            # ... and make the amplitude match that of the whole audio file:
-            audio_chunk.apply_gain(d.audio.dBFS - audio_chunk.dBFS)
+            # ... so we can cut out the corresponding audio chunk
+            # and normalize it (headroom 4db) and fade in and out ...
+            audio_chunk = normalize(
+                audio[interval.start * 1000 - 50:interval.end * 1000 + 50], 4.)\
+                .fade_in(duration=50)\
+                .fade_out(duration=50)
```

Comment:
Are there any principled reasons why this would be better than the `gain` - which attempts to make the amplitude comparable between chunks of the big file? You mentioned that some of the output was fairy noisy - so I think we should have documented reasons for any changes we apply.

Reply:
the audio chunk is an entity - all chunks should have the same loudnees while playing back regardless of the max of the entire long wav file (since an intro talk at the beginning of the entire recording has nothing to do with an elicitation) - I think

Reply:
Ok, then I think we should document that - probably as comment for a global setting `LOUDNESS` or similar.